sdk/textanalytics/azure-ai-textanalytics/src/main/java/com/azure/ai/textanalytics/AnalyzeSentimentAsyncClient.java

```diff
@@ -144,43 +141,21 @@
      * @return The {@link AnalyzeSentimentResult} to be returned by the SDK.
      */
     private AnalyzeSentimentResult convertToAnalyzeSentimentResult(DocumentSentiment documentSentiment) {
-        // Document text sentiment
-        final TextSentiment documentSentimentLabel = TextSentiment.fromString(
-            documentSentiment.getSentiment().toString());
-        if (documentSentimentLabel == null) {
-            // Not throw exception for an invalid Sentiment type because we should not skip processing the
-            // other response. It is a service issue.
-            logger.logExceptionAsWarning(
-                new RuntimeException(String.format(Locale.ROOT, "'%s' is not valid text sentiment.",
-                    documentSentiment.getSentiment())));
-        }
-
         final SentimentConfidenceScorePerLabel confidenceScorePerLabel = documentSentiment.getConfidenceScores();
-
         // Sentence text sentiment
         final List<SentenceSentiment> sentenceSentiments = documentSentiment.getSentences().stream()
             .map(sentenceSentiment -> {
-                final TextSentiment sentenceSentimentLabel = TextSentiment.fromString(
-                    sentenceSentiment.getSentiment().toString());
-                if (sentenceSentimentLabel == null) {
-                    // Not throw exception for an invalid Sentiment type because we should not skip processing the
-                    // other response. It is a service issue.
-                    logger.logExceptionAsWarning(
-                        new RuntimeException(String.format(Locale.ROOT, "'%s' is not valid text sentiment.",
-                            sentenceSentiment.getSentiment())));
-                }
                 final SentimentConfidenceScorePerLabel confidenceScorePerSentence =
                     sentenceSentiment.getConfidenceScores();
-
                 return new SentenceSentiment(sentenceSentiment.getText(),
-                    sentenceSentimentLabel,
+                    sentenceSentiment.getSentiment().toString(),
```

Comment:
We don't want to check for nulls anymore?
Initially, you were checking for null for `sentenceSentimentLabel` and `documentSentimentLabel` and throwing an exception.
Is it fine for sentimentLabel and documentSentimentLabel to be null?

Reply:
Yes. Same reason as https://github.com/Azure/azure-sdk-for-java/pull/11227/files/9ef54069c9a599fc03bf09ad1a957a4327023e31#r426102268